limitador-server/Cargo.toml

```diff
@@ -48,6 +48,7 @@ sysinfo = "0.32"
 openssl = { version = "0.10.66", features = ["vendored"] }
 metrics = "0.22.3"
 metrics-exporter-prometheus = "0.14.0"
+chrono = { version = "0.4", features = ["serde"] }
```

Comment:
If you are looking to reduce binary size further (based on strip above) you could also look at:
- lto = fat
- opt-level = s (debateable, would need benchmarking on performance)
- codegen-units=1 (this helped reduce my binary size even further)
- use default-features=false in all your dependencies, then add back in the features you need. A bit laborious, but a one-time job.